CONTRIBUTING.md

```diff
@@ -1,18 +1,20 @@
 # Contributors Guide
 
-Thank you for considering contributing to Oceananigans! This short guide will
-give you ideas on how you can contribute and help you make a contribution.
+Thank you for considering contributing to Oceananigans! 
 
 Please feel free to ask us questions and chat with us at any time if you're
 unsure about anything.
```

Comment:
Maybe add some info about how people can be reached (issues, email, slack channel)